src/openrct2/paint/tile_element/Paint.Surface.cpp

```diff
@@ -307,6 +307,14 @@ static uint32_t get_surface_image(
     const paint_session* session, uint8_t index, int32_t offset, uint8_t rotation, int32_t grassLength, bool grid,
     bool underground)
 {
+    if (!is_csg_loaded() && index >= TERRAIN_RCT2_COUNT)
+    {
+        if (index == TERRAIN_ROOF_GREY)
```

Comment:
What is this if achieving?

Reply:
If the user does not have RCT1 but loads up a park that contains RCT1 surfaces (whether via import or by loading someone else's save), this code provides a fallback to a sensible alternative. It will draw grey roofs as rock, and brown roofs, rust and logs as dirt.